pkg/services/grafana-apiserver/storage/entity/utils.go

```diff
@@ -21,6 +23,15 @@ import (
 func entityToResource(rsp *entityStore.Entity, res runtime.Object, codec runtime.Codec) error {
 	var err error
 
+	// Read the body first -- it includes old resourceVersion!
```

Comment:
I don't get how this was passing tests before -- parsing the body after setting the updated resource version made the playlist integration tests tests fail (complaining that the versions had not increased!)

Reply:
I'll have to look into that, it's possible that the body being passed here isn't properly updated with the new resource version

Reply:
ya -- this leads to a lot more questions about what should be in the body and how it gets there -- currently it is the state of the *previous* metadata with new content, then we add the new metadata on the way out